aws/synthetix/uat/web/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  rollup-full-node:
+    image: <aws_account_id>.dkr.ecr.us-east-2.amazonaws.com/optimism/rollup-full-node:synthetix-uat
+    volumes:
+      - full-node-data:/mnt/full-node:rw
+      - l1-node-data:/mnt/l1-node:rw
+    ports:
+      - 8545:8545
+    environment:
+      - CLEAR_DATA_KEY=4
+      - STARTUP_WAIT_TIMEOUT=10
+      - LOCAL_L1_NODE_PERSISTENT_DB_PATH=/mnt/l1-node
+      - L2_RPC_SERVER_PERSISTENT_DB_PATH=/mnt/full-node/level
+      - L2_WALLET_PRIVATE_KEY=0x29f3edee0ad3abf8e2699402e0e28cd6492c9be7eaab00d732a791c33552f797
```

Comment:
It doesn't matter that this is public because we're the only ones who can connect to our L2 node correct?

Reply:
Yeah, that's just the pk for the address that we deploy the contracts from. Can change if it's sensitive, but IIRC it's just the first wallet in the default ganache deployment's wallet list.

Reply:
Ultimately this and other sensitive stuff should come from AWS secrets, though.